src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -898,10 +898,8 @@ def getJobOutput(self, jobID, workingDirectory=None):
             if remoteOutput == f"{stamp}.out":
                 with open(localOutput) as f:
                     stdout = f.read()
-                os.unlink(localOutput)
```

Comment:
But it is possible that we are not using the RemoteRunner with AREX, right? So who cleans up these files then?

Reply:
Indeed, the `PilotManager` is also using `<CE>.getJobOutput()` and removes the `WorkingDirectory` (containing the outputs) at the end of the operation: https://github.com/DIRACGrid/DIRAC/blob/f877d14fb457e4da192fb5c90a82421f36678653/src/DIRAC/WorkloadManagementSystem/Service/PilotManagerHandler.py#L172

I agree that this is not ideal, but IIUC, at some point, pilots will report their outputs by themselves and I guess that the `getJobOutput` method would only be used for such a use case(?).

Reply:
OK, then there is no problem!